src/main/kotlin/org/jitsi/rtp/rtcp/rtcpfb/transport_layer_fb/tcc/RtcpFbTccPacket.kt

```diff
@@ -335,7 +340,9 @@ class RtcpFbTccPacket(
             // The packet does not contain receive deltas
             for (delta_size in delta_sizes) {
                 // Use delta sizes to detect if packet was received.
-                if (delta_size > 0) {
+                if (delta_size == 0) {
```

Comment:
after looking at this, i'm starting to think it would be more readable if we used a single constructor and named arguments in these 2 cases.  so we'd end up with:
```
if (delta_size == 0) {
    packets_.add(ReceivedPacket(seq_no.value, 0, received = false))
} else {
    packets_.add(ReceivedPacket(seq_no.value, 0, received = true))
}
```
i dunno, maybe that isn't quite it, either, but i found myself tripping over parsing difference here, since in one case we test for a delta_size of 0 and if that's false we pass a hard-coded delta size of 0.  it's making me wish we had an `UnreceivedPacket` class type, i think that'd make what's going here on most obvious.

Reply:
An `UnreceivedPacket` is fine, the problem is what to call their base class.

Maybe add `Report` to the name, so the base is `PacketReport`, with derived `UnreceivedPacketReport` and `ReceivedPacketReport`?

Reply:
that naming sgtm!